### DISCUSSION TITLE
General ND text question

### DISCUSSION THREAD

**Discussion by bdpd1** [flybywiresim/aircraft#2382] [General]
I've noticed on a few reference materials that there are often large format numeric text displayed upper right corner on the ND when radar (terrain mapping and EGPWS??) is in use (or MFD for boeing prodcuts).  Looks like an alt / FL in thousands.  What is this?

![image](https://user-images.githubusercontent.com/74915905/101986564-86228880-3c5c-11eb-9f66-06e417f58aec.png)
 
![image](https://user-images.githubusercontent.com/74915905/101986576-9b97b280-3c5c-11eb-9f5d-7eccc6d9757c.png)

**Comment by Vismaypikachu** [comment]
From what I understand from the picture, the 135* is the heading to next waypoint.  7.9Mn is distance to waypoint, and the 08:25 is the estimated time when you reach that waypoint.

**Reply by bdpd1** [reply]
Sure - on my phone so apologies. 
![EAE74684-BE04-4D71-9C55-42785419F71D](https://user-images.githubusercontent.com/74915905/101994243-f518d500-3c8e-11eb-981e-9b3fd4b3a44d.jpeg)
![46FD36B2-1C1C-48CE-82D8-5B8A9CDE151B](https://user-images.githubusercontent.com/74915905/101994244-f5b16b80-3c8e-11eb-9b6d-6678837e64ea.jpeg)

**Reply by Vismaypikachu** [reply]
let me check real quick

**Comment by Vismaypikachu** [comment]
Seems like its the lowest height of terrain and the highest
